core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -162,7 +161,11 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
     startScheduler(driver)
   }
 
-  def createCommand(offer: Offer, numCores: Int, taskId: String): CommandInfo = {
+  def createCommand(
```

Comment:
This should fit on one line as before.  I think standard style is only to split when it's > 100 chars